test/README.md

```diff
@@ -55,7 +55,7 @@ interactively follow what a test is doing, set environment variable `$TEST_SHOW_
 You can set these environment variables to configure the test suite:
 
     TEST_OS    The OS to run the tests in.  Currently supported values:
-                  "fedora-32"
+                  "fedora-32, fedora-33, rhel-8-4"
```

Comment:
Should we put `rhel-8-3` here?
@martinpitt, do we still keep `rhel-8-3` image in our CI? Thanks.

Reply:
At the moment, yes. We'll keep the image as long as any project still wants to use it. IMHO it's also convenient for manual debugging purposes to have a RHEL 8.3 image readily at hand.

Reply:
Make sense to me. Thanks!